packages/garbo/src/fights.ts

```diff
@@ -179,6 +182,11 @@ import { FreeFightQuest, runGarboQuests } from "./tasks";
 import { expectedFreeFights, possibleTentacleFights } from "./tasks/freeFight";
 import { bestMidnightAvailable } from "./resources";
 import { PostQuest } from "./tasks/post";
+import {
+  canAdventureOrUnlock,
+  canWander,
+  unlock,
+} from "garbo-lib/dist/wanderer/lib";
```

Comment:
This isn't how we should accomplish this; we should either use a new wanderer type or an existing wanderer type, or export those functions from garbo-lib through `./wanderer`